test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/Queue/QueueResponse.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+	// Common class for responses received via different Queue Types(External Resources).
+	public class QueueResponse
+	{
```

Comment:
Formatting and GetLength() - 'g"

Reply:
Formatting done. Given GetLength is a public function we are using the Pascal case as per .NET style recommendation.